Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md

```diff
@@ -0,0 +1,284 @@
+---
+slug: 2022-07-23-whats-new-in-seaquery-0.26.0
+title: What's new in SeaQuery 0.26.0
+author: SeaQL Team
+author_title: Ivan Krivosheev
+author_url: https://github.com/SeaQL
+author_image_url: https://www.sea-ql.org/SeaORM/img/SeaQL.png
+tags: [news]
+---
+
+🎉 We are pleased to release SeaQuery [`0.26`](https://github.com/SeaQL/sea-query/releases/tag/0.26.2)! Here are some feature highlights 🌟:
+
+## Dependency Upgrades
+
+[[#356](https://github.com/SeaQL/sea-query/issues/356)] We have upgraded a few major dependencies:
+- Upgrade [`sqlx`](https://github.com/launchbadge/sqlx) to 0.6
+- Upgrade [`time`](https://github.com/time-rs/time) to 0.3
+- Upgrade [`uuid`](https://github.com/uuid-rs/uuid) to 1.0
+- Upgrade [`bigdecimal`](https://github.com/akubera/bigdecimal-rs) to 0.3
+- Upgrade [`ipnetwork`](https://github.com/achanda/ipnetwork) to 0.19
+
+Note that you might need to upgrade the corresponding dependency on your application as well.
+
+## VALUES lists
+
+[[#351](https://github.com/SeaQL/sea-query/issues/350)] Add support for VALUES lists
+
+```rust
+// SELECT * FROM (VALUES (1, 'hello'), (2, 'world')) AS "x"
+let query = SelectStatement::new()
+    .expr(Expr::asterisk())
+    .from_values(vec![(1i32, "hello"), (2, "world")], Alias::new("x"))
+    .to_owned();
+
+ assert_eq!(
+     query.to_string(PostgresQueryBuilder), 
+     r#"SELECT * FROM (VALUES (1, 'hello'), (2, 'world')) AS "x""#
+ );
+```
+
+## Introduce sea-query-binder
+
+[[#273](https://github.com/SeaQL/sea-query/issues/273)] Native support SQLx without marcos
+
+```rust
+use sea_query_binder::SqlxBinder;
+
+// Create SeaQuery query with prepare SQLx
+let (sql, values) = Query::select()
+    .columns([
+        Character::Id,
+        Character::Uuid,
+        Character::Character,
+        Character::FontSize,
+        Character::Meta,
+        Character::Decimal,
+        Character::BigDecimal,
+        Character::Created,
+        Character::Inet,
+        Character::MacAddress,
+    ])
+    .from(Character::Table)
+    .order_by(Character::Id, Order::Desc)
+    .build_sqlx(PostgresQueryBuilder);
+
+// Execute query
+let rows = sqlx::query_as_with::<_, CharacterStructChrono, _>(&sql, values)
+    .fetch_all(&mut pool)
+    .await?;
+
+// Print rows
+for row in rows.iter() {
+    println!("{:?}", row);
+}
+```
+
+## SeaQL Improvements
+
+- [[#353](https://github.com/SeaQL/sea-query/pull/353)] Support `LIKE ... ESCAPE ...`  expression  
+- [[#306](https://github.com/SeaQL/sea-query/pull/306)] Move `escape` and `unescape` string to backend
+- [[#365](https://github.com/SeaQL/sea-query/pull/365)] Add method to make a column nullable
+- [[#348](https://github.com/SeaQL/sea-query/pull/348)] Add `is` & `is_not` to Expr
+- [[#349](https://github.com/SeaQL/sea-query/pull/349)] Add `CURRENT_TIMESTAMP` function
+- [[#345](https://github.com/SeaQL/sea-query/pull/345)] Add `in_tuple` method to Expr
```

Comment:
Why we have two `SeaQL Improvements` sections?

Reply:
Yep! Done)